### PR TITLE
 migrate: components_polymer3/tf_paginated_view

### PR DESCRIPTION
**Pull request by stephanwlee**

### CODE REVIEW

File: tensorboard/components_polymer3/tf_paginated_view/tf-dom-repeat.ts [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+ *       ...
+ *     </template>
+ *   </foo-comp>
+ *
+ * When implementing the behavior, the Polymer component has to invoke
+ * `this.updateDom(newItems)`. Other protected APIs are `setGetItemKey` and
+ * `setCacheSize` (please see respective doc for details).
+ */
+export class TfDomRepeat<T extends {}> extends LegacyElementMixin(
+  ArrayUpdateHelper
```
Comment (wchargin) [inline tensorboard/components_polymer3/tf_paginated_view/tf-dom-repeat.ts:48]: Hmm, interesting; I wonder how we would express a component that wanted
both the `ArrayUpdateHelper` and `DataLoaderBehavior` mixins?

File: tensorboard/components_polymer3/tf_paginated_view/tf-paginated-view-store.ts [hunk 2]
```diff
@@ -13,18 +11,15 @@
...
 
-<link rel="import" href="../tf-storage/tf-storage.html" />
+import {PolymerElement} from '@polymer/polymer';
+import {customElement} from '@polymer/decorators';
 
-<script src="paginatedViewStore.js"></script>
+import * as tf_paginated_view from './index';
 
-<script>
-  // HACK: this Polymer component allows stores to be accessible from
```
Comment (wchargin) [inline tensorboard/components_polymer3/tf_paginated_view/tf-paginated-view-store.ts:23]: :clap:

File: tensorboard/components_polymer3/tf_paginated_view/index.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+export * from './paginatedViewStore';
+export * from './tf-category-paginated-view';
+export * from './tf-dom-repeat';
```
[the remark this answers is not in the document]
Comment (stephanwlee) [reply tensorboard/components_polymer3/tf_paginated_view/index.ts:17]: Nope, that creates circular dependency. In that exact file, we do `import * as tf_paginated_view from './index'`

File: tensorboard/components_polymer3/tf_paginated_view/tf-dom-repeat.ts (hunk 1 above)
Comment (stephanwlee) [reply tensorboard/components_polymer3/tf_paginated_view/tf-dom-repeat.ts:48]: I think you do `ArrayUpdateHelper(DataLoaderBehavior(Polymer...)` or use `mixinBehaviors` property. Not sure which one is better yet.

**Pull request merged**